src/v2/api/controllers/GroupController.ts

```diff
@@ -469,38 +473,21 @@ export class GroupController {
     NoPermissionException:
       You do not have permission to perform this action`,
   })
-  async transferCaptain (
-    @Param('groupId', GroupByIdPipe) groupId: string,
-    @Body() { studentId }: SwitchCaptainDTO,
-  ) {
-    await this.groupService.switchCaptain(groupId, studentId);
-  }
-
-  @Access(PERMISSION.GROUPS_CAPTAIN_SWITCH)
-  @ApiBearerAuth()
-  @Post('/:groupId/switchCaptain')
-  @ApiOkResponse()
-  @ApiBadRequestResponse({
-    description: `\n
-    InvalidEntityIdException:
-      User with such id is not found
-      Group with such id is not found`,
+  @ApiParam({
+    name: 'groupId',
+    required: true,
+    description: 'Id of a group',
   })
-  @ApiUnauthorizedResponse({
-    description: `\n
-    UnauthorizedException:
-      Unauthorized`,
-  })
-  @ApiForbiddenResponse({
-    description: `\n
-    NoPermissionException:
-      You do not have permission to perform this action`,
+  @ApiEndpoint({
+    summary: 'Make a student a captain',
+    permissions: [PERMISSION.GROUPS_CAPTAIN_SWITCH, PERMISSION.GROUPS_$GROUPID_TRANSFER],
```

Comment:
Endpoint must work when user has 1 of these 2 permissions. But it works only when user has 2 permissions